decidim-maintainers_toolbox/README.md

```diff
@@ -0,0 +1,45 @@
+# Decidim::MaintainersToolbox
+
+Release related tools for the Decidim project.
+
+Tools for releasing, backporting, changelog generating, and working with GitHub
+
+## Installation
+
+TODO: Replace `UPDATE_WITH_YOUR_GEM_NAME_PRIOR_TO_RELEASE_TO_RUBYGEMS_ORG` with your gem name right after releasing it to RubyGems.org. Please do not do it earlier due to security reasons. Alternatively, replace this section with instructions to install your gem from git if you don't plan to release to RubyGems.org.
```

Comment:
Once this PR is approved and the gem is published in rubygems, we need to change this. 

(This placeholder was generated by the `bundle gem` command, to prevent someone squatting our rubygem name through a bot)